packages/components/src/Bottega.purs

```diff
@@ -50,10 +51,17 @@ readCreditCard :: { id :: CreditCardId, user :: UUID, paymentMethodId :: Payment
 readCreditCard creditCardObj = pure $ 
   { id: creditCardObj.id, user: creditCardObj.user, paymentMethodId: creditCardObj.paymentMethodId, maskedPan: creditCardObj.maskedPan, expiryDate: creditCardObj.expiryDate }
 
-readCreditCardRegister :: { number :: CreditCardRegisterNumber, user :: UUID, terminalUrl :: Nullable PaymentTerminalUrl, status :: { state :: String, time :: String, failReason :: Nullable String }  } -> Aff CreditCardRegister
+readCreditCardRegister :: { number :: CreditCardRegisterNumber, user :: UUID, creditCardId :: CreditCardId, paymentTerminalUrl :: Nullable PaymentTerminalUrl, status :: { state :: String, time :: String, failReason :: Nullable String }  } -> Aff CreditCardRegister
```

Comment:
A minor note: Instead of inlining this type, we could separate it to e.g. `ApiCreditCardRegister` as we are doing with the payment types: https://github.com/KSF-Media/affresco/blob/master/packages/components/src/Persona.purs#L510-L528. 
I think this is a good convetion for a better future.